translations/TablerBundle.fr.xliff

```diff
@@ -106,6 +106,26 @@
                 <source>Take me home</source>
                 <target>Retourner au début</target>
             </trans-unit>
+            <trans-unit id="Password Reset Email Sent">
+                <source>Password Reset Email Sent</source>
+                <target>Email réinitialisation de mot de passe envoyé</target>
+            </trans-unit>
+            <trans-unit id="Enter your email address and your password will be reset and emailed to you">
+                <source>Enter your email address and your password will be reset and emailed to you</source>
```

Comment:
This English doesn't make sense to me. Seems to be a 1:1 translations from Google translate. Can you explain what that Eans?